Userland/Libraries/LibGfx/AntiAliasingPainter.cpp

```diff
@@ -587,4 +651,96 @@ void AntiAliasingPainter::fill_rect_with_rounded_corners(IntRect const& a_rect,
         fill_corner(bottom_right_corner, bounding_rect.bottom_right(), bottom_right);
 }
 
+void AntiAliasingPainter::stroke_segment_intersection(FloatPoint const& current_line_a, FloatPoint const& current_line_b, FloatLine const& previous_line, Color color, float thickness)
+{
+    dbgln("stroke intersection");
```

Comment:
Left over debug output. This is printed a lot on some sites.

Reply:
Hey @Lubrsi,
good catch, i definitely will remove my debug spam and also investigate the line gaps.

Reply:
I was investigating a bit what might cause the line gaps and didn't found the right tweak to fix it. While checking with the old code base, it seems as the gaps where already there but much better covered behind some anti-aliased pixels. Good to compare with a fill color changed from "none" to "white"...